core/src/main/java/org/keycloak/sdjwt/vp/SdJwtVP.java

```diff
@@ -128,12 +130,17 @@ public static SdJwtVP of(String sdJwtString) {
         IssuerSignedJWT issuerSignedJWT = IssuerSignedJWT.fromJws(issuerSignedJWTString);
 
         ObjectNode issuerPayload = (ObjectNode) issuerSignedJWT.getPayload();
-        String hashAlgorithm = issuerPayload.get(IssuerSignedJWT.CLAIM_NAME_SD_HASH_ALGORITHM).asText();
+        String hashAlgorithm = Optional.ofNullable(issuerPayload.get(IssuerSignedJWT.CLAIM_NAME_SD_HASH_ALGORITHM))
```

Comment:
Thank you for spotting this. Actually, the spec instructs to consider the hash algorithm to be `sha-256` if not specified. I just didn't want to add logical changes to that piece of code, since it is not related to my task. 

See https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-14.html#name-hash-function-claim
> If the _sd_alg claim is not present at the top level, a default value of sha-256 MUST be used.

I updated the code in agreement with this. Would that be okay for you too?

Reply:
Thank you. It is good to replace `null` with the specification's default value `sha-256`.

Reply:
Thank you. It is good to replace `null` with the specification's default value `sha-256`.